helpers.js

```diff
@@ -1019,7 +1019,7 @@ exports.kodiTestConnection = (request, response) => {
     return request.kodi.GUI.ShowNotification(param) // eslint-disable-line new-cap
     .then((result) => {
         console.log(param.message, result);
-        response.send(param.message);
+        return response.send(param.message);
```

Comment:
`response` belongs to nodes express. the `send()` method does not return a promise or anything else. therefor its too much here :d